src/ws.cpp

```diff
@@ -187,6 +187,11 @@ static const struct wpe_bridge_interface s_wpeBridgeInterface = {
         wpe_bridge_send_connected(resource, bridgeID);
         Instance::singleton().registerSurface(bridgeID, surface);
     },
+    // disconnect
```

Comment:
This disconnect message to make an explicit wl_client_destroy makes and could to be the definitive solution to avoid crashes related with wl_client structuctures being  destroyed in `wl_client_connection_data()` (check explanation in https://github.com/Igalia/WPEBackend-fdo/pull/140) because a wayland error (fd/socket ERROR) for example, because the client is gone.

Looks good to me.